lib/image_optim/worker/svgo.rb

```diff
@@ -16,6 +17,24 @@ class Svgo < Worker
         Array(v).map(&:to_s)
       end
 
+      ALLOW_LOSSY_OPTION =
+      option(:allow_lossy, false, 'Allow precision option'){ |v| !!v }
+
+      PRECISION_OPTION =
+      option(:precision, 3, 'number of digits in the fractional part ' \
+                            '`0`..`10`, ignored in default/lossless mode') \
+                            do |v, opt_def|
+        if allow_lossy
+          OptionHelpers.limit_with_range(v.to_i, 0..10)
```

Comment:
svgo allows precision to be from [0 to 20](https://github.com/svg/svgo/blob/c372b36296c94b7cee4f77eb0af1124d384e360f/lib/svgo/coa.js#L196). I'm wondering why are negative numbers forbidden (to round integer part), but that is a question to svgo.